modules/ecr/outputs.tf

```diff
@@ -3,6 +3,11 @@ output "ecr_repo_arn_map" {
   description = "Map of image names to ARNs"
 }
 
+output "repository_host" {
```

Comment:
I'm including this upstream because the ecr repository host is used in a customer's setup for some platform configs.